src/interactive.jl

```diff
@@ -93,8 +93,10 @@ function convert_input(P::Type, ::Type{Union{T, Nothing}}, s::AbstractString) wh
 end
 
 function convert_input(P::Type, ::Type{Union{T, Symbol, Nothing}}, s::AbstractString) where T
-    return if startswith(s, ":") || startswith(s, "Symbol(\"")
-        Symbol(s)
+    # Assume inputs starting with ':' char are intended as Symbols, if a plugin accept symbols.
+    # i.e. assume the set of valid Symbols the plugin expects can be spelt starting with ':'.
+    return if startswith(s, ":")
+        Symbol(s[2:end])  # skip ':'
```

Comment:
Want to see something you will hate?
```suggestion
    return if (m=match(s, ":(.*)")) !== nothing
        Symbol(only(m))
```

Reply:
in what way would this be better? i suppose it avoids assuming `s[2:end]` will work (i.e. we could have the string `":"`) ...maybe we should switch to `chop` to avoid that? is there another rasons a regex would be better?

```suggestion
    return if startswith(s, ":")
        chop(s, head=1, tail=0)  # skip ':'
```

Reply:
(we should drop v1.0 but that's another task)

Reply:
i suppose if we wanted the string `":"` to git the string path below, not the symbol path, we could do `(m=match(s, r":(.{1,})")) !== nothing` and probably a regex is best for that behaviour? i am curious about why a regex might be preferred 😊

Reply:
oh it's not supposed to be better.
It is supposed to make you feel hate.

Reply:
Ah cool, yeah it's a lot isn't it. Not my scene